src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -41,15 +38,17 @@ export default function TrendingProjectCard({
     backgroundColor: bg,
     // Shows darker border when background is set
     border: `1px solid ${
-      bg ? 'var(--stroke-secondary)' : 'var(--stroke-tertiary)'
+      bg ? colors.stroke.secondary : colors.stroke.tertiary
     }`,
   }
 
   const rankStyle: CSSProperties = {
     fontSize: 22,
-    color: 'var(--text-primary)',
+    color: colors.text.primary,
     fontWeight: 400,
     marginRight: 15,
+    width: 25,
```

Comment:
adding width + textAlign avoids layout awkwardness from a double-digit rank number pushing over the image more than a single-digit one